src/plugins/formvalid/formvalid-server-en.hbs

```diff
@@ -28,7 +28,7 @@
 
 	{{>alertariahidden}}
 
-	<p>After submitting your page, the server returned 4 server errors. Now if you try to resubmit your form with some
+	<p>After submitting your page, the server returned 6 server errors. Now if you try to resubmit your form with some
```

Comment:
Added example so we need to increase the number.

```suggestion
	<p>After submitting your page, the server returned 7 server errors. Now if you try to resubmit your form with some
```

Reply:
Count updated.